src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceCustomRepositoryImpl.java

```diff
@@ -28,6 +28,8 @@ public class TaskResourceCustomRepositoryImpl implements TaskResourceCustomRepos
         + "%s%s%s";
 
     private static final String SELECT_CLAUSE = "SELECT t.task_id ";
+
+    private static final String DB_COL_ASSIGNEE = "assignee";
```

Comment:
Please refer to agreed naming convention for Constants -> https://hmcts-reform.slack.com/archives/C06DPL6U318/p1723714764215049